### DISCUSSION TITLE
Python Mysqlclient Error

### DISCUSSION THREAD

**Discussion by PetrJoe** [vercel/community#3071] [Help]
Please Help I am having issue with deployment of my django app in vercel server which uses external server for database.

i stumble on the error which says unable to install mysqlclient engine

**Comment by amyegan** [comment]
Hi @Akanmu01! Are you still running into this error? 

If you can share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), that will allow more people to take a closer look and identify the problem.

**Reply by PetrJoe** [reply]
Yes the error still persist

But my concern is I don't think they allow 3rd party database connection

**Comment by sabatohura** [comment]
I believe the issue might be related to the MySQL client package. I encountered a similar problem on a different server in the past, and even after moving to Vercel, the problem persisted. However, I was able to resolve it by switching to the Pymysql package.
